### PR TITLE
MDS-333 Accordion-v2

### PR DESCRIPTION
**Pull request by phgrey**
- subj
- Tooltip fix - no js, css only
- some merge_class improvement

### CODE REVIEW

File: lib/moon/design/accordion.ex [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  prop(expanded, :list, default: [])
+  prop(on_change, :event)
+  prop(single_open, :boolean, default: false)
+
+  slot(header, required: true)
+  slot(content, required: true)
+
+  def handle_event("on_change_default", _params = %{"value" => index}, socket) do
+    # IO.puts("ON_CHANGE_ACCORDION ############ params = #{inspect(_params, pretty: true)}")
```
Comment (margusp2heathmont) [inline lib/moon/design/accordion.ex:22]: Please not comments in codebase.

File: lib/moon/helpers/styled.ex [hunk 2]
```diff
@@ -45,4 +45,10 @@ defmodule Moon.Helpers.Styled do
       "#{x}: var(--#{y})"
     end
```
Comment (margusp2heathmont) [inline lib/moon/helpers/styled.ex:46]: All this module should be deleted.

File: lib/moon_web/pages/design/accordion_page.ex [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+            <Accordion.Header>Ipsum</Accordion.Header>
+            <Accordion.Content>{ipsum()}</Accordion.Content>
+            <Accordion.Header>Dolor</Accordion.Header>
+            <Accordion.Content>{dolor()}</Accordion.Content>
+          </Accordion>
+        </:example>
+        <:code>{get_example_1_code()}</:code>
+      </ExampleAndCode>
+
+      <ExampleAndCode title="One item open at a time and Disabled" {...rndid()}>
```
Comment (margusp2heathmont) [inline lib/moon_web/pages/design/accordion_page.ex:125]: This will update on each render?

Comment (margusp2heathmont) [reply lib/moon_web/pages/design/accordion_page.ex:125]: Not ideal behaviour.

**Review by margusp2heathmont — CHANGES_REQUESTED** [review]
Please see comments.

File: lib/moon_web/pages/design/accordion_page.ex (hunk 3 above)
Comment (phgrey) [reply lib/moon_web/pages/design/accordion_page.ex:125]: thanks, haven't thought about

**Review by tkullisaar — APPROVED** [review]

**Review by margusp2heathmont — APPROVED** [review]

**Pull request merged**